src/Generic/Data/Internal/Enum.hs

```diff
@@ -21,6 +21,7 @@
 module Generic.Data.Internal.Enum where
```

Comment:
Can you also make the start of the comment above "Generic deriving for 'Enum' and 'Ix'."

Reply:
and Bounded?

Reply:
I think one file for this is okay, all three parts are pretty small and not bound to change wildly.

Reply:
Yeah add Bounded too!